lib/fake_sqs/actions/receive_message.rb

```diff
@@ -27,6 +33,7 @@ def call(queue_name, params)
               xml.ReceiptHandle receipt
               xml.MD5OfBody message.md5
               xml.Body message.body
```

Comment:
Most likely this is also the reason for your commented out spec below, but you're missing here:
`xml.MD5OfMessageAttributes message.message_attributes_md5`